### PR TITLE
Adding a new reminder

### PR DESCRIPTION
**Pull request by becs919**
@martensonbj @Tman22 (if you have any advice on the date problem, what would be great!)
When user is on root, they click a create button and it takes them to reminders/new
Added to list of reminders as you type the title
After clicking submit, you can see the body but NOT THE DATE??
closes partially issue #4 

## Purpose
We allowed the user to add a reminder (with exception of date!) and have it render on the page!  

## Approach
We first made a button that takes the user to the reminders/new page. Then with the help of Taylor, got our inputs and actions together to save a reminder and render on the page. 

### Learning
Lot's of learning for this one! Thanks Brenna for the docs you send over, also Taylor for all of your help! 

### Open Questions and Pre-Merge TODOs
-Need to figure out why the date isn't capturing 
-Need to clear input fields on click 

### Test coverage 
Added two acceptance tests for new. Passing! 

### Follow-up tasks
Finish getting date to capture as well as clearing input fields on click

and issue #5 

### Screenshots
after

### CODE REVIEW

File: app/templates/reminders/new.hbs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<div class='add-new-reminder'>
+  {{input type='text' value=model.title placeholder='Enter a title'}}
+  {{log model.title}}
```
Comment (Tman22) [inline app/templates/reminders/new.hbs]: Take this out

File: tests/acceptance/new-test.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+import Ember from 'ember';
+
+moduleForAcceptance('Acceptance | new');
+
+test('visiting /new', function(assert) {
+  server.createList('reminder', 5);
+
+  visit('reminders/new')
+
```
Comment (Tman22) [inline tests/acceptance/new-test.js]: Can you fix the spacing to be consistent here.

File: tests/acceptance/new-test.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  andThen(function() {
+    assert.equal(currentURL(), '/reminders/new')
+  });
+});
+
+test('adding a new reminder', function(assert) {
+  server.createList('reminder', 5);
+
+  visit('reminders/new')
```
Comment (Tman22) [inline tests/acceptance/new-test.js]: Semi-colons?

**Comment by becs919** [comment]
@Tman22 @martensonbj 
Deleted semi colons, spacing, and log per Taylor's request. We still aren't rendering the date though for this issue!

**Comment by becs919** [comment]
@Tman22 @martensonbj 
We got the date to save and render on the page with this last commit. Now we only have to figure out how to clear the fields after submit.

**Comment by martensonbj** [comment]
Nice work guys! in terms of clearing the fields, think about your controller/component as the "internal state" of your template. Check out this example from [ember-groceries](https://github.com/turingschool-examples/ember-groceries/blob/master/app/components/add-grocery.js) - notice in particular how after the record is created you call `setProperties()` and reset the properties you are targeting to empty strings.  

@becs919

**Comment by becs919** [comment]
@Tman22 @martensonbj 

We finished issue 4 and can close #4 

Check it out!

**Comment by martensonbj** [comment]
Good work team! Way to push through.

**Pull request merged**